core/src/main/scala/better/files/Implicits.scala

```diff
@@ -115,14 +115,17 @@ trait Implicits extends ManagedResource.FlatMap.Implicits with Scanner.Read.Impl
         bufferSize: Int = DefaultBufferSize
       ): ObjectInputStream =
       new ObjectInputStream(if (bufferSize <= 0) in else buffered(bufferSize)) {
-        override protected def resolveClass(objectStreamClass: ObjectStreamClass): Class[_] =
+        override protected def resolveClass(_objectStreamClass: ObjectStreamClass | Null): Class[_] = {
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.